### ISSUE TITLE
Move Packages in Assets Subfolder

### ISSUE THREAD

**Issue by luizcarlosfx** [modesttree/Projeny#79]
I think it should be possible to move packages folder out of the root Assets folder. Why isn't it possible? I like to move all my exensions in a folder called "_Extensions", I also have two assets in the asset store and they both use the same root folder.

**Comment by svermeulen** [comment]
Yeah now that you mention it, it would not be difficult to control explicitly where every package gets placed in the project.  We'd just need to change the project configuration file to support custom directories.  Right now the project config looks like this:

```
AssetsFolder:
- Foo
PluginsFolder:
- Bar
- Qux
- Gorp
```

We could change it to this:

```
Packages:
- Foo
- Bar
- Qux
- Gorp
```

And then interpret any nested dictionary underneath 'Packages' to be a sub directory, to allow for this to mirror the first example:

```
Packages:
- Foo
- Plugins:
    - Bar
    - Qux
    - Gorp
```

Or more complex stuff like:

```
Packages:
- Gorp
- CustomDir:
    - Foo
- Plugins:
    - Bar
    - OtherCustomDir:
        - Qux
```

**Comment by dugsteen** [comment]
I was searching around after adding Issue #80* and realized that this may solve my problem too; as long as the package names could be disassociated with their folder names. In other words, I'd need a way to make sure both my Projeny projects 'FirebaseAuth' and 'FirebaseAnalytics' could be placed in an 'Assets\Firebase' folder. (And of course I wouldn't want one to overwrite the other)

\* TL;DR: Can't include Google Firebase or Facebook add-ons as Projeny packages b/c they rely on various files being placed in specifically named folders.
